index.html

```diff
@@ -137,7 +141,7 @@ <h2 class="work-title section-title">My Recent Works</h2>
               <div class="card">
```

Comment:
- [ ] [Optional] Additionally, I would recommend using the `<article>` tag instead of the `<div>` tag to represent a card. The `<article>` tag is more semantically meaningful in this context. You can refer to [this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) to learn more about it. 🙏 Please make the necessary changes accordingly. Keep up the good work! 👍